client/js/components/statement/DpStatementAnonymize.vue

```diff
@@ -59,8 +59,8 @@
       <div
         v-show="actions.anonymizeStatementText"
         class="u-ml">
-        <dp-tooltip-icon
-          icon="fa-question-circle float-right"
+        <dp-contextual-help
+          icon="question float-right"
```

Comment:
The `icon` prop of DpContextualHelp defaults to `question`, so this can be removed here.

`float-right` has to go into the `class` attr.

Reply:
Done here [be33bc2](https://github.com/demos-europe/demosplan-core/pull/2376/commits/be33bc2d4b3ad5fce405dfbb868049c07e0673ee)w